fvm/src/kernel/default.rs

```diff
@@ -245,6 +256,14 @@ where
     }
 }
 
+impl<C> Validator for DefaultKernel<C>
```

Comment:
Do we need a trait for this? It looks like a private method for the default kernel?